README.md

```diff
@@ -87,6 +87,11 @@ End tags may be included (if done consistently) and attributes may be quoted (us
 though the prevelant theme is to omit end tags and not quote attributes (unless they contain a
 space).
 
+### Merge policy
+
+If you can commit to this repository, see the
+[maintainer guidelines](https://github.com/whatwg/meta/blob/master/MAINTAINERS.md).
```

Comment:
Do we want `WHATWG maintainer guidelines` for consistency?

Reply:
Hmm maybe. If we did that we should say "WHATWG Maintainer Guidelines" I suppose, just like "WHATWG Code of Conduct". I guess I could update any outstanding PRs and fix it for the remainder if something else "Meta" needs updating in the future.

Reply:
... if something else "Meta" needs updating in the future. - @annevk 

I see what you did there ;-)